### ISSUE TITLE
Web App on Linux 'Runtime Stack' dropdown is wrong format

### ISSUE THREAD

**Issue by krispenner** [microsoft/azure-pipelines-tasks#12514]
## Required Information

**Question, Bug, or Feature?**  
*Type*: Bug

**Enter Task Name**: Azure App Service deploy v4

## Environment
- Server - Azure Pipelines
    
- Agent - Hosted 
    
## Issue Description

The actual text shown in the Runtime Stack dropdown is saved to the app service. However this format is invalid and causes the container to select PHP in my case and not run my app.

I have to use an Azure PowerShell Task to then update it correctly afterwards.

The UI does not let me enter the text I want either, it just ends up replacing my text of 'DOTNETCORE|2.2' with the list item of '2.2 (DOTNETCORE|2.2)'.

![image](https://user-images.githubusercontent.com/8837910/76391914-be8fe180-632d-11ea-93f7-fb22ac161a19.png)

Here the wrong format is saved to the app service, I can also see this value in the Resource Explorer, while the Portal itself shows blank as it has it is invalid for it's values as well.

![image](https://user-images.githubusercontent.com/8837910/76392004-f6972480-632d-11ea-869c-80e420703e51.png)

**Comment by SumiranAgg** [comment]
@krispenner The display value and value used to set as runtime stack is different. We do take care to set the correct format via task. See snippet below:
![image](https://user-images.githubusercontent.com/8316321/76725765-46824b00-6775-11ea-86a4-1a207b40dc74.png)

Can you share the task name and version you are using to pinpoint the issue. You will find something like this in your task logs:
![image](https://user-images.githubusercontent.com/8316321/76725846-7c273400-6775-11ea-846f-709ea1130a53.png)

**Comment by SumiranAgg** [comment]
Closing the issue. If you are still seeing the issue please provide information asked above and re-open the issue.

**Comment by krispenner** [comment]
@SumiranAgg sorry, I missed your previous reply, here are the details you are looking for.

Version 4.163.3

![image](https://user-images.githubusercontent.com/8837910/77188460-4646c000-6a93-11ea-88f0-81c32be14450.png)

I realized later that this is not a required field and as I had already set it correctly during my app service creation script I left this blank and so it worked properly for me because the task was no longer setting it at all.

I've also now moved on to multi-stage YAML pipelines so I no longer use this UI either.
